app/rest/dao/src/test/resources/logback-test.xml

```diff
@@ -27,7 +27,8 @@
   </appender>
 
   <root level="INFO">
-    <appender-ref ref="STDOUT" />
+    <!--<appender-ref ref="STDOUT"/>-->
```

Comment:
Is this a deliberate change? We're now not seeing test log in build output which is kinda a pain to debug, going to file, even when running tests in IDE.

Reply:
I do not remember to have changed it deliberately but for dao, there's almost no log in any test except the for the Hibernate Validator initialization. The only test that does some logging is ReadApiClientDataTest but it does soon System.out so I guess I did it to suppress Hibernate Validator noises.